fastembed/text/onnx_embedding.py

```diff
@@ -0,0 +1,307 @@
+import os
+from multiprocessing import get_all_start_methods
+from typing import List, Dict, Any, Tuple, Union, Iterable, Type
+
+import numpy as np
+import onnxruntime as ort
+
+from fastembed.common.model_management import locate_model_file
+from fastembed.common.models import load_tokenizer, normalize
+from fastembed.common.utils import define_cache_dir, iter_batch
+from fastembed.parallel_processor import ParallelWorkerPool, Worker
+from fastembed.text.text_embedding_base import TextEmbeddingBase
+
+supported_flag_models = [
```

Comment:
```suggestion
supported_onnx_models = [
```

We'll add the GTE and Sentence Transformers models here as well. Wouldn't want to overfit to BGE/Flag

Reply:
:+1: